### PR TITLE
Bury people in asteroid sand

### PR DESCRIPTION
**Pull request by Crystalwarrior**
You can now bury people in sand! Dig out a hole and drag&drop someone into it. Enjoy your space funerals.
Renamed /obj/item/weapon/ore/glass into /obj/item/weapon/ore/sand since it makes wayyyyy more fucking sense that way.
Added  /proc/deleteAllInList(list/L) - qdels all datums in list L.

### CODE REVIEW

File: code/modules/mining/mine_turfs.dm [hunk 1]
```diff
@@ -589,19 +590,83 @@ var/global/list/rockTurfEdgeCache
...
+	new/obj/item/weapon/ore/sand(src)
 	dug = 1
 	icon_plating = "asteroid_dug"
 	icon_state = "asteroid_dug"
 	return
 
+/turf/simulated/floor/plating/asteroid/MouseDrop_T(atom/movable/target, mob/user)
+	if(!dug)
+		return
+	if(user.stat || user.lying || !Adjacent(user) || !target.Adjacent(user))
```
Comment (JohnGinnane) [inline code/modules/mining/mine_turfs.dm:642]: Would checking for user.stat mean you can't bury people alive?

Comment (TDcoolguy300) [inline code/modules/mining/mine_turfs.dm:642]: user.stat should just be some generic status check, such as being stunned. It should still work

Comment (JohnGinnane) [inline code/modules/mining/mine_turfs.dm:642]: Do you know where I can find a list of statuses?

Comment (TDcoolguy300) [inline code/modules/mining/mine_turfs.dm:642]: Shrug

Comment (danhibiki) [inline code/modules/mining/mine_turfs.dm:642]: https://github.com/HippieStationCode/HippieStation13/blob/7e8e4e70be2bf67c1be5800ef7fd0f66ff50efd6/code/__DEFINES/stat.dm#L5

File: code/modules/mining/mine_turfs.dm [hunk 2]
```diff
@@ -589,19 +590,83 @@ var/global/list/rockTurfEdgeCache
...
+	new/obj/item/weapon/ore/sand(src)
+	new/obj/item/weapon/ore/sand(src)
+	new/obj/item/weapon/ore/sand(src)
+	new/obj/item/weapon/ore/sand(src)
 	dug = 1
 	icon_plating = "asteroid_dug"
 	icon_state = "asteroid_dug"
 	return
 
+/turf/simulated/floor/plating/asteroid/MouseDrop_T(atom/movable/target, mob/user)
```
Comment (Carbonhell) [inline code/modules/mining/mine_turfs.dm:639]: Iunno wouldn't it be better to Buckle the mob to the turf and then deal with it from there

Comment (Crystalwarrior) [inline code/modules/mining/mine_turfs.dm:639]: ehhhhhhh, I do it the sleeper way. kind of.
ALSO Buckling is not supported for turfs I'm 100% sure.

File: code/modules/mining/mine_turfs.dm (hunk 1 above)
Comment (JohnGinnane) [inline code/modules/mining/mine_turfs.dm:642]: Cheers

**Comment by myontasm** [comment]
This is probably gonna be left unused after introduction but yeah sure why not :+1:

**Comment by Lawligagger** [comment]
alrighty

**Comment by Crystalwarrior** [comment]
Oooooh one quick mention, you can also bury a single item instead.

**Comment by myontasm** [comment]
The bury site best be obvious otherwise people will just hide sensitive items in the sand.

**Pull request merged**